client-ui/src/app/model/lineage-graph/leneage-graph.models.ts

```diff
@@ -20,12 +20,12 @@ import { Stylesheet } from 'cytoscape';
 export const LINE_WIDTH_PLANE = 10;
```

Comment:
I just noticed this file is called **leneage**-graph.models.ts insted of **lineage**-graph.models.ts